src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyoutItem.cs

```diff
@@ -73,5 +139,685 @@ internal void InvokeClick()
 			Click?.Invoke(this, new RoutedEventArgs(this));
 			Command.ExecuteIfPossible(this.CommandParameter);
 		}
+
+		public MenuFlyoutItem()
+		{
+			m_bIsPointerOver = false;
+			m_bIsPressed = false;
+			m_bIsPointerLeftButtonDown = false;
+			m_bIsSpaceOrEnterKeyDown = false;
+			m_bIsNavigationAcceptOrGamepadAKeyDown = false;
+			m_shouldPerformActions = false;
+
+			Initialize();
+		}
+
+		// Prepares object's state
+		void Initialize()
+		{
+			Loaded += (s, e) => ClearStateFlags();
+
+			this.RegisterDisposablePropertyChangedCallback((s, e, args) => OnPropertyChanged2(args));
+		}
+
+		// Apply a template to the 
+
+		protected override void OnApplyTemplate()
+		{
+			base.OnApplyTemplate();
+
+			TextBlock keyboardAcceleratorTextBlock = this.GetTemplateChild("KeyboardAcceleratorTextBlock") as TextBlock;
+			m_tpKeyboardAcceleratorTextBlock = keyboardAcceleratorTextBlock;
+
+			SuppressIsEnabled(false);
+			UpdateCanExecute();
+
+			// Sync the logical and visual states of the control
+			UpdateVisualState();
+		}
+
+		// PointerPressed event handler.
+		protected override void OnPointerPressed(PointerRoutedEventArgs pArgs)
+		{
+			base.OnPointerPressed(pArgs);
+			var handled = pArgs.Handled;
+			if (!handled)
+			{
+				PointerPoint spPointerPoint;
+				PointerPointProperties spPointerProperties;
+				spPointerPoint = pArgs.GetCurrentPoint(this);
+				spPointerProperties = spPointerPoint.Properties;
+				var bIsLeftButtonPressed = spPointerProperties.IsLeftButtonPressed;
+				if (bIsLeftButtonPressed)
+				{
+					m_bIsPointerLeftButtonDown = true;
+					m_bIsPressed = true;
+
+					pArgs.Handled = true;
+					UpdateVisualState();
+				}
+			}
+		}
+
+		// PointerReleased event handler.
+		protected override void OnPointerReleased(PointerRoutedEventArgs pArgs)
+		{
+			base.OnPointerReleased(pArgs);
+
+			bool handled = false;
+
+			handled = pArgs.Handled;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'pArgs' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)